pkg/config/config.go

```diff
@@ -1067,7 +1068,7 @@ func load(config Config, origin string, loadSecret bool) (*Warnings, error) {
 	SanitizeAPIKeyConfig(config, "api_key")
 	// Environment feature detection needs to run before applying override funcs
 	// as it may provide such overrides
-	detectFeatures()
+	DetectFeatures()
```

Comment:
Why was this made public? It is still defined in the same package as far as I can see?

Reply:
It's used in integration tests